src/components/Calculator.js

```diff
@@ -0,0 +1,206 @@
+import React, { Component } from 'react';
+import calculate from './logic/calculator';
+import Navbar from './pages/Navbar';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: '',
+      next: null,
+      operation: null,
+    };
+  }
+
+  handleClick = (e) => {
+    const data = e.target.value;
+    const state = calculate(this.state, data);
+    this.setState(state);
+  };
+
+  render() {
+    const { total, next, operation } = this.state;
+    return (
+      <div>
+        <Navbar />
+        <div className="calc">
+          <div className="displayScreen">
+            {total}
+            {operation}
+            {next}
```

Comment:
- [ ] Please kindly refactor the Calculator to be a functional component and use React hooks